website/server/models/group.js

```diff
@@ -985,6 +995,31 @@ schema.methods.removeTask = async function groupRemoveTask (task) {
   }, {multi: true}).exec();
 };
 
+// Returns true if the user has reached the spam message limit
+schema.methods.checkChatSpam = function groupCheckChatSpam (user) {
+  if (this._id !== TAVERN_ID) {
+    return false;
+  } else if (user.contributor && user.contributor.level >= SPAM_MIN_EXEMPT_CONTRIB_LEVEL) {
```

Comment:
Yea, these can be just broken into two if statements, but it is a style preference.